### PR TITLE
Add sprites, text

### LINKED ISSUE #20: Styling and Site Design
- [x] Find Planet Images
- [x] Create Color Scheme
- [x] Test Colors for color blind accessibility 
- [x] Test Colors for Contrast
- [x] Find Font

### PR DESCRIPTION
**Pull request by ConnorAndersonLarson**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ ] Fix
- [ ] Feature
- [ ] Refactor
- [x] Style
- [ ] Testing

### Are there any known bugs?

- [x] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Find/Add fonts for webpage (Orbitron and Raleway for Primary/Secondary fonts)
Find/Add planet and other space themed sprites, credit creator on page.
Add credit for Earth gif on page
Create/Test Color Scheme for project, add to documentation in project board:
Background Color:
 * Dark Blue: #071927
Large Font/Accent Colors : 
* Mid Blue: #288FDE
* Red: #F04528
All Font/Accent Colors : 
* Light Blue: #68CCD7
* Yellow: #F5AE45
* Light Gray: #D0D7DF

### What ticket(s) do the changes resolve?
Close #20 

Please Link Issues to this PR as necessary.
#20 

### Notes for the Reviewer
Reviewer can check contrast of color palette using the following contrast checker
[Individual Contrast](https://webaim.org/resources/contrastchecker/)
Otherwise Check images under public>planets and public>space 

### How to Test Changes?
Site should show new font/color as well as credits on page. 

### Next Iterations
Implement planet images with planetarium component.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,7 +1,22 @@
+* {
+  font-family: 'Orbitron', sans-serif;
+  color: #D0D7DF;
+}
+
+p {
+  font-family: 'Raleway', sans-serif;
+}
+
```
Comment (alexthompson207) [inline src/App.css:9]: I like the look of these fonts

File: public/index.html [hunk 2]
```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Orbitron:wght@400;500;600;700;800;900&family=Raleway:ital,wght@0,200;0,300;0,400;1,200;1,300;1,400&display=swap" rel="stylesheet">
```
Comment (knees4bees) [inline public/index.html:7]: Love these fonts!

File: src/App.tsx [hunk 3]
```diff
@@ -31,12 +31,16 @@ class App extends React.Component<{}, AllData> {
...
         </header>
         <main>
+
           <h2>SortBox</h2>
           <h2>Planetarium</h2>
         </main>
+        <footer>
+          <p className="credits">Icons made by <a href="https://www.flaticon.com/authors/monkik" title="monkik">monkik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></p>
+          <p className="credits">Gif made by <a href="https://www.catchmaj.com/">Cat Chmaj</a> at <a href="https://giphy.com/gifs/VI2UC13hwWin1MIfmi">GIPHY</a></p>
+        </footer>
```
Comment (alexthompson207) [inline src/App.tsx:43]: Nice work with credits

File: pull_request_template.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# PR Rancid Tomatillos
+# PR PlanetParty
```
Comment (knees4bees) [inline pull_request_template.md:1]: 😆

**Review by alexthompson207 — COMMENTED** [review]
Great job gathering all the images and stylings!

File: src/App.tsx [hunk 5]
```diff
@@ -31,12 +31,16 @@ class App extends React.Component<{}, AllData> {
...
           <h1 className="title">Planet Party!</h1>
-          <img className="galaxy" src="galaxy2.png" alt="galaxy" />
         </header>
         <main>
+
           <h2>SortBox</h2>
           <h2>Planetarium</h2>
         </main>
+        <footer>
+          <p className="credits">Icons made by <a href="https://www.flaticon.com/authors/monkik" title="monkik">monkik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></p>
```
Comment (knees4bees) [inline src/App.tsx:41]: Awesome idea to add the image credits.

**Review by knees4bees — APPROVED** [review]
Looks good! Literally.

**Pull request merged**